library/src/main/java/io/ona/kujaku/manager/DrawingManager.java

```diff
@@ -278,6 +285,20 @@ private Polygon stopDrawing() {
         setDrawing(false);
         setCurrentCircle(null);
 
+        Polygon polygon = getCurrentPolygon();
+
+        // Delete annotations
+        this.deleteAll();
+        // Refresh
+        this.refresh(false);
+
+        return polygon;
+    }
+
+    /**
+     * @return Returns the up to date drawn polygon at any point in time
+     */
+    public Polygon getCurrentPolygon(){
```

Comment:
I am curious, what's the use case of exposing this and using it as it is?

Reply:
@ekigamba I am measuring the boundary area as it's getting drawn and showing it to the user

Reply:
Makes sense